spec/syntax.md

```diff
@@ -705,6 +705,18 @@ on the pairing, ordering, or contents of _markup_ during _formatting_.
 Examples in this section are meant to be illustrative and
 might not match future requirements or usage.
 
+During the specification's "tech preview" phase,
+implementations SHOULD NOT implement functionality using function or markup options
+that might later be replaced by attributes or some other solution.
```

Comment:
I think it's perhaps the other way around? We should encourage implementations to experiment with alternatives (under their namespace), and then share feedback?

What is the end goal of including this recommendation?